searchcore/src/vespa/searchcore/proton/server/proton.h

```diff
@@ -198,7 +197,8 @@ class Proton : public IProtonConfigurerOwner,
     int64_t getConfigGeneration();
 
     size_t getNumDocs() const;
-    size_t getNumActiveDocs() const;
+    // Active (searchable), and targetActive that will be searchable when idealstate is reached
+    std::pair<size_t, size_t> getNumActiveDocs() const;
```

Comment:
@baldersheim Consider adding a struct for this, to avoid confusion of what is _active_ and _target_active_.